### ISSUE TITLE
DCMP Video Example CSV File

### ISSUE THREAD

**Issue by clapierre** [benetech/ImageShare#72]
Attached is the original DCMP Video CSV file.
I will work with PAC to formalize the column names so that we can create a CSV to JSON script to allow for bulk importing of resources into Imageshare.

This ticket will get closed once the final format is established and a new attachment is provided conforming to that standard CSV template.

**Comment by clapierre** [comment]
[DCMP_ImageShare_original.csv.txt](https://github.com/benetech/Imageshare/files/4577299/DCMP_ImageShare_original.csv.txt)

Github didn't support .csv file so renamed it .txt

**Comment by jkva** [comment]
Related: https://github.com/benetech/Imageshare/issues/71#issuecomment-627131725

**Comment by clapierre** [comment]
We will need to create a new DCMP export of all the videos matching our new schema and create an associated JSON manifest to bulk import these videos.

**Comment by sinabahram** [comment]
Can we assign people/timeline to this?

**Comment by clapierre** [comment]
@theladymay will be looking into getting this done.
I hope this will get done by the end of this month.

**Comment by sinabahram** [comment]
How are we doing with this one @clapierre ?

**Comment by clapierre** [comment]
We did include 2 Video DCMP examples in our master CSV file so we can be sure external resources hosted by DCMP is handled correctly and exercises the "length" of a file.

We are currently working on organizing the entire excel spreadsheet of over 2700 videos, which has repeated records we must remove and organize the "topics & sub topics" which have and place one overarching "subject" and the rest go into Keywords.  My daughter is doing this work.  We also have to get alt-text descriptions for all these featured images which they don't have.  So I suspect we won't have the entire DCMP library to import for a couple weeks.  We did have a meeting with them and they are willing to work with us to help us reduce the amount of work moving forward as they continue to add new videos.
